packages/core/strapi/lib/services/metrics/admin-user-hash.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const hash = require('hash.js');
```

Comment:
Also, any reason why we're not using node's crypto standard package here?

```js
const crypto = require('crypto');

return crypto.createHash('sha256').update(ctx.state.user.email).digest('hex');
```

Reply:
We use hash.js on admin code, so I wanted to be consistent

Reply:
@ivanThePleasant Just out of curiosity: where are we using this package? I can't find it anywhere ...